src/main/java/thrift/logic/Logic.java

```diff
@@ -35,9 +38,24 @@ CommandResult execute(String commandText, TransactionListPanel transactionListPa
      */
     ReadOnlyThrift getThrift();
 
-    /** Returns an unmodifiable view of the filtered list of transactions*/
+    /** Returns if the given command requires a refresh of the filteredlist. */
+    boolean requireFilteredListRefresh(Command command);
```

Comment:
Boolean methods should be named to sound like booleans.

Reply:
Changed to `isRefreshingFilteredList`